monkey/infection_monkey/i_puppet/i_puppet.py

```diff
@@ -125,6 +125,7 @@ def exploit_host(
         name: str,
         host: VictimHost,
         current_depth: int,
+        servers: Sequence[str],
```

Comment:
Why are we adding this in a PR meant for mypy fixes?

Reply:
Because all implementations of `exploit_host` pass `servers` parameter, but the interface doesn't have it. That causes a mypy error.